test/jasmine/tests/waterfall_test.js

```diff
@@ -1460,31 +1522,31 @@ describe('waterfall hover', function() {
             .then(function() {
                 var out = _hover(gd, 0, 1000.5, 'closest');
                 expect(out.yLabelVal).toEqual(1002.201);
-                expect(out.extraText).toEqual(undefined);
+                expect(out.extraText).toEqual('Final: $1,002.201m<br>$2.2m ▲<br>Initial: $1,000.001m');
```

Comment:
Why did these assertions change?

Reply:
We did not fill any extra text for blue bars (totals). But it is pretty useful to display this info for them for example when there is a `base` other than zero.

Reply:
Ok, but what does _initial_ and _final_ refer to for total bars?

Reply:
The base and top of the bar.

Reply:
I don't think base/top here represent _inital_ and _final_ values.

Reply:
Could be displayed in [codepen after](https://codepen.io/MojtabaSamimi/pen/KjdxgR?editors=0010).

Reply:
I agree that initial and final don't make sense in 'total' cases